lib/room.rb

```diff
@@ -1,12 +1,25 @@
-class Rom
+class Room
+
+attr_reader :category,
+            :painted
```

Comment:
Since we have a method `is_painted?` to return the current state of `@painted`, we don't need this attr_reader.